platform/ios/Makefile

```diff
@@ -12,30 +16,26 @@ else
 xcodebuild := xcodebuild
 endif
 
+.PHONY: build
 lockxcodeversion:
 	sudo xcode-select -switch /Applications/Xcode_13.2.1.app
 
-# Github secrets
-DISTRIBUTION_CERTIFICATE = $POLYPOD_APPLE_IOS_APPSTORE_P12_FILE_BASE64
-DISTRIBUTION_CERTIFICATE_PWD = $POLYPOD_APPLE_IOS_APPSTORE_P12_PASS
-DISTRIBUTION_PROVISIONING_PROFILE = $POLYPOD_APPLE_IOS_APPSTORE_PROVISION_PROFILE_BASE64
-APPSTORE_CONNECT_API_KEY = $POLYPOD_APPLE_IOS_APPSTORE_API_PRIVATE_KEY_BASE64
-APPSTORE_API_KEY_ID = $POLYPOD_APPLE_IOS_APPSTORE_API_KEY_ID
-APPSTORE_API_ISSUER_ID = $POLYPOD_APPLE_IOS_APPSTORE_API_ISSUER_ID
-
+.PHONY: build
```

Comment:
Any reason why this is repeated?

Reply:
It's just a stylistic choice, pretty common though. I'm mostly following this one: https://clarkgrubb.com/makefile-style-guide#phony-targets